tools/Deployment/gulpfile.js

```diff
@@ -209,9 +217,9 @@ gulp.task("syncBranchCore", () => {
 gulp.task("test", gulp.series("clean", "build", "e2eTest", "publish:myget-test"));
 gulp.task("dev", gulp.series("clean", "build", "e2eTest"));
 gulp.task("stable", gulp.series("clean", "build", "e2eTest", "publish:myget-dev"));
-gulp.task("master:build", gulp.series("clean", "build:release", "e2eTest", "packAssetZip", "updateGhPage"));
 
-gulp.task("master", gulp.series("master:build", "publish:myget-master", "publish:chocolatey", "publish:gh-release"));
+gulp.task("master:build", gulp.series("clean", "build:release", "e2eTest"));
```

Comment:
move "publish:gh-release" from "master:release" to "master:build"?

Reply:
Why?
1. It is a release step
3.It will update the tag,which will block the myget and Choco release

Reply:
sorry... I mean updateGhPage

Reply:
Why?
1. It is a release step
2.It will update the tag,which will block the myget and Choco release

---
In reply to: [193930886](https://github.com/dotnet/docfx/pull/2860#discussion_r193930886) [](ancestors = 193930886)